src/components/status/TilAlleOppfolgingsplanerKnapp.tsx

```diff
@@ -0,0 +1,16 @@
+import { Back } from "@navikt/ds-icons";
+import { Button } from "@navikt/ds-react";
+import { useLandingUrl } from "hooks/routeHooks";
+import NextLink from "next/link";
+
+export const TilAlleOppfolgingsplanerKnapp = () => {
```

Comment:
TilForside, evnt?

Reply:
Jeg renamet den til `TilLandingssideKnapp` nå. Siden vi kalte katalogen under components for `landing`, så valgte jeg det.